frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx

```diff
@@ -173,9 +178,18 @@ const RoleBindingPermissionsTableRow: React.FC<RoleBindingPermissionsTableRowPro
                 />
               </SplitItem>
             </Split>
+          ) : isDefaultGroup ? (
+            <Tooltip content="The default group always has access to model registry">
```

Comment:
full stop missing at the end according to [design](https://www.figma.com/proto/KTsYhANYQqB0NUrXW7wQH1/Model-Registry?page-id=6782%3A296930&node-id=8709-219955&viewport=1846%2C28%2C0.43&t=anh0eGvvgwFUHiMb-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=8709%3A221017)

Reply:
Good catch! Added.